### DISCUSSION TITLE
Making the Fancy User Interface on ESP Has Never Been Easier! · Espressif Developer Portal

### DISCUSSION THREAD

**Discussion by giscus** [espressif/developer-portal#287] [General]
# Making the Fancy User Interface on ESP Has Never Been Easier! · Espressif Developer Portal

The developer resources in just one place!

https://developer.espressif.com/blog/making-the-fancy-user-interface-on-esp-has-never-been-easier/

**Comment by vsltech** [comment]
https://youtu.be/aEsFvHppP6E?si=fVp3fNxZQzyH4ovC

We implemented lvgl in many of our embedded design services
